spec/system/partners/managing_requests_system_spec.rb

```diff
@@ -168,6 +188,38 @@
         end
       end
 
+      context 'WHEN a request has invalid values' do
+        before do
+          fill_in 'Comments', with: Faker::Lorem.paragraph
+
+          # Select items
+          3.times do |idx|
+            if idx != 0
+              click_link 'Add Another Item'
+            end
+
+            last_row = find_all('tr').last
+            last_row.find_all('.form-control').last.fill_in(with: 1)
+          end
+
+          click_button 'Submit Essentials Request'
+        end
+
+        it 'should show an error message with the instructions ' do
+          expect(page).to have_content('Oops! Something went wrong with your Request')
+          expect(page).to have_content('Ensure each line item has a item selected AND a quantity greater than 0.')
+          expect(page).to have_content('Still need help? Submit a support ticket here and we will do our best to follow up with you via email.')
+        end
+
+        it "should show invalid values in the form" do
```

Comment:
Can we combine these 2? System specs are expensive, so let's get all our expectations together :)

Reply:
@dorner do you still feel this way? or can this PR proceed? I would also like to rebase this PR on top of #4099 cause they have some overlap

Reply:
Yep, since we're addressing it in a followup PR I'm OK with it.